app/certificate_expiry_check.py

```diff
@@ -98,10 +66,22 @@ def send_email(email_type, params):
 
 
 def main():
+    # Instantiate services
```

Comment:
The comments here are mainly just for me to visualise processes, and will be removed in the future